TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/console_link/console_link/models/cluster.py

```diff
@@ -48,15 +66,24 @@ def __init__(self, config: Dict) -> None:
         self.endpoint = config["endpoint"]
         if self.endpoint.startswith("https"):
             self.allow_insecure = config.get("allow_insecure", False)
-        if "authorization" in config:
+        try:
             self.auth_type = AuthMethod[config["authorization"]["type"].upper()]
+        except KeyError:
+            self.auth_type = None
```

Comment:
There _shouldn't_ be a possibility of a KeyError here because we just validated the schema and `authorization` and `authorization.type` are both required

Reply:
You're right, updated